tests/python/contrib/test_ethosu/infra.py

```diff
@@ -193,6 +193,7 @@ def create_test_runner(accel="ethos-u55-256"):
     file_dir = os.path.dirname(os.path.abspath(__file__))
     test_root = os.path.join(file_dir, "reference_system")
     ethosu_macs = accel[accel.rfind("-") + 1 :]
+    ethosu_variant = accel[accel.find("-") + 1 : -4].upper()
```

Comment:
Feel free to ignore, just thought this might be slightly clearer :)
```suggestion
    _, ethosu_variant, ethosu_macs = accel.split("-")
    ethosu_variant = ethosu_variant.upper()
```

Reply:
Yes, that looks much nicer, I'll change it in the next patch, thanks @lhutton1!